x/gov/keeper/test_common.go

```diff
@@ -153,7 +153,7 @@ func createTestInput(t *testing.T, isCheckTx bool, initPower int64) (sdk.Context
 		AddRoute(types.RouterKey, types.ProposalHandler)
 
 	keeper := NewKeeper(cdc, keyGov, pk.Subspace(types.DefaultParamspace).WithKeyTable(types.ParamKeyTable()),
```

Comment:
Generally I find these to be more readable when the follow the following structure: 

```go
foo := SomeFuncWithManyArgs(
  arg, arg, ...,
)
```

Reply:
should we set a line length limit? for the entire repo? this is within the current line limit 120

Reply:
I think we've discussed this before to great disagreement. I'm OK with the cap being 120.